test/e2e/app-dir/ppr-navigations/incremental/incremental.test.ts

```diff
@@ -15,15 +17,22 @@ async function getDotNextFiles(next: NextInstance): Promise<Array<string>> {
 }
 
 describe('ppr-navigations incremental', () => {
-  const { next, isNextDev, isTurbopack } = nextTestSetup({
+  const { next, isNextDev, isTurbopack, isNextDeploy } = nextTestSetup({
     files: __dirname,
   })
 
   it('can navigate between all the links and back without writing to disk', async () => {
-    const before = !isNextDev && !isTurbopack ? await getDotNextFiles(next) : []
+    const before =
+      !isNextDev && !isTurbopack && !isNextDeploy
+        ? await getDotNextFiles(next)
+        : []
 
     const browser = await next.browser('/')
 
+    await browser.waitForIdleNetwork()
+    await waitForHydration(browser)
+    await setTimeout(500)
```

Comment:
Do we still need to wait for 500 if we waited for hydrated above?

Reply:
Seemingly, yes. The hydration event only fires once on full page navigation, and not after client navigations. I'm thinking that the test failures we're seeing here is actually related to the prefetches and not hydration, but until I have more context this seems to make the tests pass consistently.